mediacapture-streams/obtaining-local-multimedia-content/navigatorusermedia/deny.html

```diff
@@ -26,8 +26,13 @@ <h1 class="instructions">Description</h1>
     t.done();
   }),
   t.step_func(function (error) {
-    assert_equals(error.name, "SecurityError", "SecurityError returned");
-    assert_equals(error.constraintName, null, "constraintName attribute not set for permission denied");
+    assert_equals(error.name, "securityError", "securityError returned");
+    // NOTE ALEX: this is also nullable, but practically, all the browsers but FF set it to an empty string.
```

Comment:
we should test based on what the spec says, not based on what implementations do; if and when the spec gets proven to be wrong, we will update the test, but if we hide implementation bugs in our test cases, we won't find the discrepancies between spec and implementations.